### PR TITLE
design doc: add design doc

### PR DESCRIPTION
**Pull request by ajeddeloh**
Accidently `git push -d` instead of `git push -f` on the other PR, which made it unopenable.

Please check for:

  -  Grammar, clarity, etc
  -  That I accurately recorded the discussion
  -  Formatting seems good

I only did the partitioning issue. We can add summaries of other closed issues in more PRs.

I envision that when we want to close out an issue, we PR this doc with a summary. Once we agree the PR is good and accurately encapsulates the discussion, we merge the PR and close the issue.

As the doc grows we may want to break it out into multiple docs in a directory, but that can come later.

Finally, I'd prefer to keep this doc about design of FCOS itself and not about the process around it. We can have a seperate doc for that (the README does a great job right now) if we want.

### CODE REVIEW

File: Design.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Design Document
+
+This document captures the design decisions made after dicussion in issues. When a design issue is closed, the
+conclusion should be summarized here with a link to the issue.
+
+## Disk Layout
+
+Originally discussed in issue #18
```
Comment (dustymabe) [inline Design.md]: would be nice if this was a hyperlink to #18 - when I view it rendered it doesn't seem to auto hyperlink

Comment (ajeddeloh) [reply Design.md]: sure, I thought it would autolink, but I guess not?

File: Design.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+would involve downloading the OSTree repo contents and doing the deploy between the Ignition disks and files stage if
+the root filesystem has changed. This is currently untested.
+
+### Open Questions:
+
+ - What do we do about 4k sector disks? We could make a "hybrid" disk image, but it technically breaks the GPT spec and
+   may not work with poorly implemeted UEFIs.
+ - What is the exact partition layout?
+ - Do we make /etc a ro bind mount?
+ - Do filesystem do we use for / and /var? ext4?
```
Comment (dustymabe) [inline Design.md]: "What filesystem do we use"

Comment (ajeddeloh) [reply Design.md]: Oops. Fixed.

**Comment by dustymabe** [comment]
LGTM

**Comment by dustymabe** [comment]
anybody else want to review ?

**Comment by ajeddeloh** [comment]
I'll give it a day and merge tomorrow unless someone else has other comments. SG? We should probably decide how we decide merge-readiness for these.

File: Design.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Design Document
+
+This document captures the design decisions made after dicussion in issues. When a design issue is closed, the
```
Comment (bgilbert) [inline Design.md]: `discussion`

File: Design.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+conclusion should be summarized here with a link to the issue.
+
+## Disk Layout
+
+Originally discussed in issue [#18](https://github.com/coreos/fedora-coreos-tracker/issues/18). See also [dustymabe's comment](https://github.com/coreos/fedora-coreos-tracker/issues/18#issuecomment-409668929) summarizing the discussion in the FCOS meeting.
+
+### Summary:
+
+ - FCOS will use a "dd-able" image and ship with a standard parition layout.
```
Comment (bgilbert) [inline Design.md]: partition

**Pull request merged**